mobile/app/src/main/java/com/mylook/mylook/notifications/NotificationCenter.java

```diff
@@ -100,6 +105,10 @@ private void updateNotifications(){
     @Override
     public void onBackPressed() {
         super.onBackPressed();
-        finish();
+        if (fromDeepLink){
+            Intent intent= new Intent(getApplicationContext(), MainActivity.class);
+            startActivity(intent);
+            //finish();
```

Comment:
con un this.finish() no anduvo?

Reply:
Nop, porque al ser un child activity busca ir parent (Todas las recomendaciones) y da null pointer por todos lados @GisiGimenez